test/verify/machineslib.py

```diff
@@ -1083,7 +1089,8 @@ def installWithErrorTest(dialog):
                                                    memory_size=900, memory_size_unit='GiB',
                                                    storage_size=10, storage_size_unit='MiB',
                                                    os_vendor=config.APPLE_VENDOR,
-                                                   os_name=config.MACOS_X_LEOPARD))
+                                                   os_name=config.MACOS_X_LEOPARD,
+                                                   connection='session'))
```

Comment:
'system' was the previous default, right? I. e. the new thing here is that it makes it possible to create session VMs as user.

Could this please also get a tests case that succeeds? This only covers a failing one.

Reply:
There was no hard coded default. If ```virsh uri``` reported ```qemu:///system``` the system connection would have been used else the session.
Looks like admin user on the test images has default ```qemu:///session``` uri so all VMs previously got created under session connection.
So, I 'll have to change some test to use system instead. Failed and Successful.
Thanks for the heads up here.